test/intfuncs.jl

```diff
@@ -487,3 +487,17 @@ end
     ci = code_typed(() -> 14 // 21)[][1]
     @test ci.code == Any[Core.ReturnNode(2 // 3)]
 end
+@testset "binomial" begin
+    for T in (Int8, UInt16, Int32, UInt64)
+        for x in rand(-isqrt(typemax(T)):isqrt(typemax(T)), 1000)
```

Comment:
That doesn't work for the unsigned types I think. Maybe
```suggestion
        for x in rand(T(-isqrt(-widen(typemin(T)))):isqrt(typemax(T)), 1000)
```
would do it, but... uh...
Might be clearer to branch depending on whether `T` is signed or unsigned.